include/grpcpp/passive_listener.h

```diff
@@ -21,6 +21,7 @@
 namespace grpc_core {
 class ListenerInterface;
```

Comment:
I don't think we need any of these forward declarations anymore.

Reply:
Removed.